recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
Can't actually import as we need to set `DRMAA_LIBRARY_PATH` to an install of `libdrmaa.so`. Also would need an install of SGE and have `SGE_ROOT` and `SGE_CELL` set. So, instead we just verify that we can import, but will get the expected exception. This verifies it is doing what it should.

More sophisticated testing is unnecessary as `drmaa` is pure Python and goes through CI anyways.